### ISSUE TITLE
Update to package required

### ISSUE THREAD

**Issue by spatialhusky** [munterfinger/hereR#81]
Hi,

I've been receiving error 400 while trying to use your package. After digging around and asking Here's customer service, there seems to be an update required to the function's URL script piece.

![image](https://user-images.githubusercontent.com/58966082/98048918-48784780-1df4-11eb-81ed-79e05156910d.png)

This image is a response from their team. Was wondering if there is any quick way to push an update here 👍

**Comment by munterfinger** [comment]
Hi, thanks for reporting.

This issue is already fixed in #73. 
Can you update to **hereR 0.5.1** from CRAN and try again?

```r
install.packages("hereR")
```

**Comment by spatialhusky** [comment]
Thanks for the quick response, yes, after physically removing the package from my computer and adding again the issue was solved.

Also, on that address I input on that test, the one weird thing is that in appearance it shows the result in Australia, when the original address expects at least a result in the same country (USA). While I do realize the output of the geocode in the URL includes several options, I'm guessing it takes the first result.

Really appreciate your package, I find incredibly useful to be honest! But would love an ability to pull multiple results from one geocoded string (like an option saying pull first, last or all for each result), and also the ability to pull the geocoding score to have a numeric view of how well the geocoding did.

Would love to collaborate and learn to do this myself if you don't have the bandwidth to tackle this options :)

Thanks again, and incredibly gratefull for your contribution, regards from Costa Rica

**Comment by munterfinger** [comment]
Thank you for the positive feedback on the package! I am glad that the problem was fixed by updating to the current version.

Concerning your feature request I have opened a new issue #83 and will therefore close this issue.

Since you exposed your APIKEY in the screenshot, I would recommend to re-generate the key on HERE's developer portal to prevent from use by others.

Regards from Switzerland
